test/jdk/TEST.groups

```diff
@@ -661,4 +662,8 @@ jdk_containers_extended = \
 
 jdk_core_no_security = \
    :jdk_core \
-   -:jdk_security
\ No newline at end of file
+   -:jdk_security
+
+# Set of tests for `@since` checks in source code documentation
+jdk_since_check = \
+   tools/sincechecker/testjavabase/CheckSince_javaBase.java
```

Comment:
Generally, the symbol of a red line in a red circle is indicative of a missing final newline at the end of the file.
It is recommend that files end with exactly one newline character.